ddtrace/contrib/internal/aws_lambda/patch.py

```diff
@@ -12,7 +12,7 @@
 from ddtrace.internal.wrapping import unwrap
 from ddtrace.internal.wrapping import wrap
 from ddtrace.trace import tracer
-
+from ....settings._config import _get_config
```

Comment:
```suggestion
from ddtrace.settings._config import _get_config
```
Though maybe the relative import is necessary in the context of a lambda function. Is that the case?

Reply:
No I don't believe so